lib/membrane_opus/parser.ex

```diff
@@ -116,11 +130,27 @@ defmodule Membrane.Opus.Parser do
           channels :: 0..2
         ) ::
           {:ok, remaining_buffer :: binary, pts :: Membrane.Time.t(), packets :: [Buffer.t()],
-           channels :: 0..2}
+           channels :: 0..2, generate_best_effort_timestamps? :: boolean()}
           | :error
-  defp maybe_parse(data, pts, input_delimitted?, processor, packets \\ [], channels \\ 0)
-
-  defp maybe_parse(data, pts, input_delimitted?, processor, packets, channels)
+  defp maybe_parse(
+         data,
+         pts,
+         input_delimitted?,
+         processor,
+         packets \\ [],
+         channels \\ 0,
+         generate_best_effort_timestamps?
```

Comment:
I see that `maybe_parse` function doesn't use `generate_best_effort_timestamps?`. This argument is only passed to the recurrent calls of this function